code/game/objects/structures/window_spawner.dm

```diff
@@ -39,6 +39,13 @@
 			return
 		activate()
 		return INITIALIZE_HINT_QDEL
+	else
+		if(activated)
+			return
+		activate()
+		spawn(10)
+			qdel(src)
```

Comment:
Why not `return INITIALIZE_HINT_QDEL`?

Reply:
because it works iffy with submap post-initialization